### ISSUE TITLE
i have an example of this with rollup if you're interested

### ISSUE THREAD

**Issue by ezekielchentnik** [developit/preact-redux-example#4]

**Comment by developit** [comment]
Absolutely. Want to publish it somewhere?  I'd be happy to link to it and recommend people go that route for smaller bundle sizes.

**Comment by diorahman** [comment]
Can we have it? 🙏

**Comment by diorahman** [comment]
Can we have it? 🙏

**Comment by Gufsky1** [comment]
@diorahman you can check out this https://github.com/ezekielchentnik/preact-pwa

although, I want to make sure it's useful to you.  'preact-pwa' is very opinionated (towards perf), and may not be the normal 'rollup, preact, redux' solution you may expect;  I can absolutely provide that for you if that helps!

**Comment by ezekielchentnik** [comment]
@diorahman you can check out this https://github.com/ezekielchentnik/preact-pwa

although, I want to make sure it's useful to you.  'preact-pwa' is very opinionated (towards perf), and may not be the normal 'rollup, preact, redux' solution you may expect;  I can absolutely provide that for you if that helps!
